src/token/erc1155/erc1155_receiver.cairo

```diff
@@ -61,7 +62,7 @@ mod ERC1155ReceiverComponent {
         +HasComponent<TContractState>,
         +SRC5Component::HasComponent<TContractState>,
         +Drop<TContractState>
-    > of IERC1155ReceiverCamel<ComponentState<TContractState>> {
+    > of interface::IERC1155ReceiverCamel<ComponentState<TContractState>> {
```

Comment:
any reason to make this explicit here?

Reply:
The mixin requires another import from `erc1155::interface`. I figured it's better to just use the single `interface` mod rather than bringing three interfaces into scope. No strong preference though

Reply:
but didn't the previous version imported only the ones it used? I lean towards dropping interface:: from these impl definitions to improve readability

Reply:
Done!